### DISCUSSION TITLE
在gpio_control_block的檔案裡出現concurrent assignment error

### DISCUSSION THREAD

**Discussion by Josephood7** [bol-edu/HLS-SOC-Discussions#167] [Q&A] [answered]
跑vivado simulation的時候 在gpio_control_block的檔案裡出現concurrent assignment的error
不知道是哪邊的程式或是設定有問題
有人也遇過這個問題ㄇ
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/144367420/993fae7f-6719-4de9-bc8b-133e8767aeac)

**Comment by junshiuanhsieh** [comment, marked as answer]
`default_nettype 從none改成wire 應該可以解決

**Reply by Josephood7** [reply]
謝謝你~ 的確是這個問題
